test/org/zaproxy/zap/spider/parser/SpiderHtmlFormParserUnitTest.java

```diff
@@ -903,11 +913,12 @@ public void shouldIgnoreBaseHtmlIfActionIsAbsoluteWhenParsingPostForm() {
     @Test
     public void shouldSetValuesToFieldsWithNoValueWhenParsingGetForm() {
```

Comment:
It seems `definedValues` is not initialised before being used leading to an exception (maybe that's already fixed locally?). Once that's fixed the tests still fail, they should still pass without changing them though.
The problem seems to be that `field.getValues()` is no longer being used, before it obtained/used the first value when available and non-empty.